spec/requests/evaluation_forms_spec.rb

```diff
@@ -136,7 +133,6 @@
 
         expect(evaluation_form.closing_date).to eq(new_closing_date)
         # Disallow other updates while still allowing end_date
-        expect(evaluation_form.title).to eq(old_title)
```

Comment:
instead of removing this, let's preserve the expectation that other data isn't updated by try to change the instructions instead and showing the original value is retained